helm/ingress-azure/templates/deployment.yaml

```diff
@@ -30,6 +30,34 @@ spec:
       - name: {{ .Chart.Name }}
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        readinessProbe:
+          httpGet:
+            path: /health/ready
+        {{- if .Values.kubernetes }}
+        {{- if .Values.kubernetes.healthProbeServicePort }}
+            port: {{ .Values.kubernetes.healthProbeServicePort }}
```

Comment:
give `Values.kubernetes.healthProbeServicePort` a default value in values.yml ?

Reply:
Good idea! Thanks!